doc/plugin_server_keymanager_aws_kms.md

```diff
@@ -42,6 +44,75 @@ The IAM role must have an attached policy with the following permissions:
 - `kms:UpdateAlias`
 - `kms:DeleteAlias`
 
+
+### Key policy
+The plugin can generate keys using a default key policy or it can load and use a user defined policy.
+
+#### Default key policy
+The default policy used by SPIRE depends on whether the SPIRE server has assumed a role or not.
+
+If SPIRE server has assumed a role (recommended), the keys are created by default with the following policy:
```

Comment:
I think this needs to be updated to reflect the fact that role will be required in the future. I think it is OK to state here that role is mandatory (and remove the second part of this section) since that's the direction we're going in

Reply:
I updated it to reflect that, please let me know if you think it can be improved  🙂